docs/grafana-datasource.md

```diff
@@ -143,7 +153,7 @@ docker-compose -f docker-compose.yaml up
 When Grafana starts successfully datasources should be present on the datasources tab
 
 <p>
-  <img src="grafana-datasource_provision_datasources.webp" width="800" alt="Configuration">
```

Comment:
@hagen1778 , @zekker6 , @Loori-R , it looks like the synced docs at https://github.com/VictoriaMetrics/grafana-datasource/ are outdated comparing to the docs in VictoriaMetrics repository. Please update the docs at https://github.com/VictoriaMetrics/grafana-datasource/ in accordance to 334a739ff6fca9c3886db0b0c63ab84b1138a079 and https://github.com/VictoriaMetrics/VictoriaMetrics/blob/32c064a401dde6ac310379b02fa9ebaccb8e9b1f/docs/assets/README.md

Reply:
See also https://github.com/VictoriaMetrics/VictoriaMetrics/blob/4ac7e3a355b6dbac1867efca72acc057f653120b/docs/Makefile and https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#images-in-documentation

Reply:
let me fix

Reply:
fixed